CommandLine/XblConnectedStorage/Program.cs

```diff
@@ -78,11 +84,11 @@ private static async Task<int> Main(string[] args)
 
         private static async Task<int> OnDownload(Options options)
         {
-            ulong xuid = options.XboxUserId;
+            string gamertag = options.GamerTag;
```

Comment:
make sure spaces work.  aka "2 Dev 1234" which are comment test account names.

Reply:
Tested and working